tests/packages/rabbitmqadmin.bats

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bats
+
+# Requires RabbitMQ service to be running
+
+setup() {
+  mkdir -p "${HOME}/bin"
+}
+
+teardown() {
+  rm $HOME/bin/rabbitmqadmin
+}
+
+@test "[rabbitmqadmin.sh] Installs successfully" {
+  run source /dev/stdin <<< "$(cat ./packages/rabbitmqadmin.sh)"
```

Comment:
I wouldn't `source` the script, simply calling it is fine in this case (since we don't need the script to modify the current shell)